src/amber/router/context.cr

```diff
@@ -122,6 +122,12 @@ class HTTP::Server::Context
   end
 
   protected def finalize_response
+    if Amber.settings.http_keep_alive
```

Comment:
Let's make this a `KeepAlive` pipe instead.

Reply:
The only real issue I see with this is that keep-alive needs to be enabled globally to do any good not just in one pipeline.